packages/core/src/node/cluster/master-process.ts

```diff
@@ -8,13 +8,20 @@
 import { EventEmitter } from 'events';
 import { ServerWorker } from './server-worker';
 
-export type MasterProcessEvent = 'started' | 'restarted' | 'restarting';
+export type MasterProcessEvent = 'started' | 'restarted' | 'restarting' | 'exited';
 export class MasterProcess extends EventEmitter {
 
     protected serverWorker: ServerWorker | undefined;
+    protected isRestarting: boolean = false;
```

Comment:
`restarting`, `is` for method names

Reply:
`restarting` is used to return some kind of promise, but it is for a ponctual event and it is not a lock / flag, which we would need here ?

Tell me if I missed something again.

Reply:
it is minor, just about naming, if you add a method returning `isRestarting`, you will need to name it differently since it will collide with a property, because of it a property usually called `restarting` and access method `isRestarting`.